frontend/pfiniture/src/components/landing/SellerListings.js

```diff
@@ -0,0 +1,133 @@
+
+    useEffect(() => {
+        if (user) {
+            setUser(props.userInfo)
+            setListings(props.listings)
+        } else {
+            setUser({})
+            setListings([])
+        }
+    }, [props.userInfo, props.listings])
+
+    return (
+        <div>
+        {!listings
+            ? "No Matched Results"
+            :
+            listings.map((listing, index) => (
+                <div key={index} className={classes.root}>
+                    <Paper className={classes.gridPaper}>
+                        <Grid
+                            container
+                            spacing={2}
+                            direction="row"
+                            justifyContent="center"
+                            alignItems="center">
+                            <Grid item>
+                                <ButtonBase className={classes.image}>
+                                    <img
+                                        className={classes.img}
+                                        alt={listing.furniture.name}
+                                        src={listing.images[0] ? listing.images[0].imageUrl : ""} />
+                                </ButtonBase>
+                            </Grid>
+                            <Grid item xs={12} sm container>
+                                <Grid item xs container direction="column" spacing={2}>
+                                    <Grid item xs>
+                                        <Typography gutterBottom variant="h6" component="h2">
+                                            {listing.furniture.name}
+                                        </Typography>
+                                        <Typography variant="body2" gutterBottom>
+                                            {listing.description}
+                                        </Typography>
+                                        <Typography variant="body2" color="textSecondary">
+                                            Type: {listing.type}
+                                        </Typography>
+                                    </Grid>
+                                    <Grid item>
+                                        <Button size="small" color="primary" onClick={() => onAddToCart(listing.id)}>
+                                            Add to Cart
+                                        </Button>
+                                    </Grid>
+                                </Grid>
+                                <Grid item>
+                                    <Typography variant="subtitle1">${listing.furniture.price}</Typography>
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                    </Paper>
+                </div>
+            ))
+        }
+        </div>
+    )
+}
+
+function onAddToCart(listingID) {
+    const userID = "6104918f9a92da1084fb7438";
```

Comment:
Was this used for testing? The user ID is hardcoded

Reply:
This is was part of @Rahman2600 's code for checkout that I didn't touch. I've already made note to change this in the cart/payment system that I'll be working on today

Reply:
Awesome, thanks for the clarification!